### PR TITLE
dist/tools/testrunner: fix flake8 remaining issues

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR make the testrunner script compliant with RIOT's [Python coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#python-coding-convention).

### Issues/PRs references

None

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ dist: trusty
 
 before_install:
   - sudo apt-get install coreutils realpath doxygen graphviz python-lesscpy cppcheck coccinelle pcregrep python3-flake8
+  - which flake8
```
Comment (gebart) [inline .travis.yml]: I assume that you are experimenting on Travis with these commits.

Comment (aabadie) [reply .travis.yml]: yes, it doesn't work because python 2 flake8. I'm trying to understand what's going on there.

Comment (gebart) [reply .travis.yml]: see https://packages.ubuntu.com/trusty/python3-flake8 and https://packages.ubuntu.com/trusty/amd64/python3-flake8/filelist for some hints

Comment (aabadie) [reply .travis.yml]: Thanks, it helped. Actually python3-flake8 has a dependency to python-flake8. The python2 flake8 was taken into account and that's why there was a issue with `ProcessLookupError`: it only exists with python3.

Installing flake8 using pip3 fixes the problem. That's certainly why I did that this way at the very beginning of the work on adding flake8 check to static tests.

**Comment by gebart** [comment]
Is it not possible to avoid pip3 and use the distribution provided flake8 package in some way?

**Comment by aabadie** [comment]
> the distribution provided flake8 package in some way

this was the case before but python3-flake8 also installs python-flake8 and the latter is taken into account by travis. I don't know how this can be avoided on travis. The simple solution is to use pip3.

**Comment by aabadie** [comment]
All CI are green, 2 ACKs, go!

**Pull request merged**